backend/tests/test_llm_providers_api.py

```
@@ -0,0 +1,124 @@
+                filtered = self.parent_items
+                stmt_str = str(stmt).lower()
+                if "where" in stmt_str and "llm_providers.id =" in stmt_str:
+                    # super hacky mock for tests
+                    try:
+                        import re
+                        match = re.search(r"llm_providers.id = :id_1", stmt_str)
+                        if match:
+                            # We can just assume it's getting the last created provider id for these tests
+                            pass
+                    except Exception:
+                        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)